### ISSUE TITLE
Namespace timeout explicit option for kuttl

### ISSUE THREAD

**Issue by cap1984** [kudobuilder/kuttl#454]
**What would you like to be added**: namespace deletion specific timeout argument

**Why is this needed**: The latest 0.15.0 release has been introduced the strict namespace removal policy which I prefer very much. However, since it's not a part of any TestStep, we have to use TestSuite wide timeout parameter and it's not the perfect use-case to follow. I believe more granular control for namespace removal action is a way to go further.
Thanks.

**Comment by iblancasa** [comment]
I like this request. It can be really useful for tests where a lot of stuff is created.

**Comment by eddycharly** [comment]
Aren’t created resources removed individually ?
If so the namespace removal should not be long.

**Comment by cap1984** [comment]
> Aren’t created resources removed individually ? If so the namespace removal should not be long.

Nope, as a user I just want to remove all created resources at once without any extra efforts.

**Comment by eddycharly** [comment]
This is especially problematic when using finalizers.

**Comment by cap1984** [comment]
> This is especially problematic when using finalizers.

That is why I dropping them on the last step :D

**Comment by eddycharly** [comment]
But it won't work =)
